google/cloud/spanner/value_test.cc

```diff
@@ -945,13 +945,12 @@ INSTANTIATE_TEST_SUITE_P(
         std::make_tuple("NaNFloat64", Value(std::nan("NaN")), "NaN"),
         std::make_tuple("String", Value("Seatac Astronomy"),
                         "Seatac Astronomy"),
-        std::make_tuple("Bytes", Value("DEADBEEF"), "DEADBEEF"),
+        std::make_tuple("Bytes", Value(Bytes("DEADBEEF")), "DEADBEEF"),
```

Comment:
Perhaps BYTES should render with the `b` prefix (and the quotes) ... `b"DEADBEEF"`.